samples/.eslintrc.yml

```diff
@@ -1,4 +1,4 @@
 ---
 rules:
   no-console: off
-  node/no-missing-require: off
+  no-warning-comments: off
```

Comment:
This is ok to have on somewhat globally in samples.  It hides warnings generated from `TODO:` statements we use in the snippets.